pkg/cli/testdata/expected_release_output.txt

```diff
@@ -61,6 +63,8 @@ OSS Suite release:
 - [cyberark/conjur](#cyberarkconjur)
 - [cyberark/conjur-oss-helm-chart](#cyberarkconjur-oss-helm-chart)
 - [cyberark/conjur-api-python3](#cyberarkconjur-api-python3)
+- [cyberark/conjur-api-java](#cyberarkconjur-api-java)
+- [cyberark/conjur-api-go](#cyberarkconjur-api-go)
```

Comment:
Wondering about these - pinged in slack

Reply:
yup, from there:
> I added these to test when CL is not given or is “unknown”

otherwise there is nothing (afaict) that tests what happens in those flows